tests/test_ops.py

```diff
@@ -288,21 +294,34 @@ def test_tensordot(arr3d, storage, opt):
     assert_equal(actual.todense(), expected)
 
 
-def test_matmul(arr2d, arr3d, opt):
-    A_finch = finch.Tensor(arr2d)
-    B_finch = finch.Tensor(arr2d.T)
-    C_finch = finch.permute_dims(A_finch, (1, 0))
-    D_finch = finch.Tensor(arr3d)
+@pytest.mark.parametrize(
```

Comment:
Just to double check, these new tests are a superset of the old ones, right?

Reply:
Correct, other than the one you point out below for a dimension mismatch. I'll add a separate test for that one.